Assets/MixedRealityToolkit-Examples/UX/Scripts/LaunchDialogScript.cs

```diff
@@ -8,13 +11,37 @@ namespace MixedRealityToolkit.Examples.UX
 {
     public class LaunchDialogScript : Interactive
     {
-        public GameObject resultText;
-
         [SerializeField]
         private Dialog dialogPrefab;
+        private bool isDialogLaunched;
+        private GameObject resultText;
```

Comment:
if this was public before, do we want to also serialize the field to users can reference the text result object in the inspector?